src/scripting/Commands.sp

```diff
@@ -35,6 +35,7 @@ stock void InitializeCommands()
 
 	ConVar_MTF2MaxRounds = CreateConVar("mtf2_maxrounds", "4", "Sets the maximum rounds to be played. 0 = no limit (not recommended).", 0, true, 0.0);
 	ConVar_MTF2IntermissionEnabled = CreateConVar("mtf2_intermission_enabled", "1", "Controls whether or not intermission is to be held half way through the maximum round count. Having Intermission enabled assumes you have a intermission integration enabled - for example the SourceMod Mapchooser integration.", 0, true, 0.0, true, 1.0);
+	ConVar_MTF2BonusPoints = CreateConVar("mtf2_bonuspoints", "1", "Controls whether or not minigames should have a bonus point.", 0, true, 0.0, true, 1.0);
```

Comment:
Could you change the default value of the convar to be 0? It would be better that this is opt-in than being enabled by default.

Reply:
done